lib/view.js

```diff
@@ -95,6 +95,24 @@ export default class AdvancedOpenFileView {
         });
     }
 
+    moveCursorTop() {
+        this.content.setState((state, props) => {
```

Comment:
The callback form of `setState` is meant for when you want to modify the state in a way that depends on the current value of the state, like adding to a number in the state. In this case, we just want to set the index to 0, so we can just pass it in directly:

```js
this.content.setState({selectedIndex: 0});
```